cmake/OpenCVFindWebP.cmake

```diff
@@ -21,13 +21,15 @@ else()
 
     # Look for the library.
     FIND_LIBRARY(WEBP_LIBRARY NAMES webp)
-    MARK_AS_ADVANCED(WEBP_LIBRARY)
+    FIND_LIBRARY(WEBP_MUX_LIBRARY NAMES webpmux)
+    FIND_LIBRARY(WEBP_DEMUX_LIBRARY NAMES webpdemux)
 
     # handle the QUIETLY and REQUIRED arguments and set WEBP_FOUND to TRUE if
     # all listed variables are TRUE
     INCLUDE(${CMAKE_ROOT}/Modules/FindPackageHandleStandardArgs.cmake)
     FIND_PACKAGE_HANDLE_STANDARD_ARGS(WebP DEFAULT_MSG WEBP_LIBRARY WEBP_INCLUDE_DIR)
-
-    SET(WEBP_LIBRARIES ${WEBP_LIBRARY})
+    
+    SET(WEBP_LIBRARIES ${WEBP_LIBRARY} webpmux webpdemux)
```

Comment:
You should only add webpmux and webpdemux if they are found. When building WebP from third_party, it is all bundled in one library.
So:
```cmake
if(WEBP_MUX_LIBRARY)
  SET(WEBP_LIBRARIES ${WEBP_LIBRARY} ${WEBP_MUX_LIBRARY})
endif()
```